### DISCUSSION TITLE
is there a guide on how to restore ETCD errors? Especially after hard reset?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#23812] [Q&A and General discussion]
is there a guide on how to restore ETCD errors?

**Comment by yhmo** [comment]
What do you mean etcd errors? Etcd service panic by out of storage or out of memory?

**Reply by martinenkoEduard** [reply]
service panic after a hard reset (especially after power out).

**Reply by yhmo** [reply]
> service panic after a hard reset (especially after power out).

Just restart milvus cluster. Etcd/Pulsar/MinIO are independent services, they can recover themselves. If these services can recover successfully, then milvus can restart successfully.
